src/beast/beast/module/core/native/linux_Files.cpp

```diff
@@ -201,8 +200,8 @@ class DirectoryIterator::NativeIterator::Pimpl
     DIR* dir;
 };
 
-DirectoryIterator::NativeIterator::NativeIterator (const File& directory, const String& wildCard)
-    : pimpl (new DirectoryIterator::NativeIterator::Pimpl (directory, wildCard))
+DirectoryIterator::NativeIterator::NativeIterator (const File& directory, const String& wildCard_)
+    : pimpl (new DirectoryIterator::NativeIterator::Pimpl (directory, wildCard_))
```

Comment:
@scottschurr Your expectation is incorrect:
"The names of non-public data members and formal arguments used to initialize public data members have a trailing underscore."